#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_header_fix.dpatch by  <root@intrepid-builder64.kristjansson.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adds missing headers

@DPATCH@

diff -ru fuppes-0+svn578/src/lib/HTTP/HTTPMessage.cpp fuppes-0+svn578-fixed/src/lib/HTTP/HTTPMessage.cpp
--- fuppes-0+svn578/src/lib/HTTP/HTTPMessage.cpp	2008-07-09 20:01:02.000000000 -0400
+++ fuppes-0+svn578-fixed/src/lib/HTTP/HTTPMessage.cpp	2008-07-09 19:58:32.000000000 -0400
@@ -27,6 +27,7 @@
 #include "../SharedConfig.h"
 #include "HTTPParser.h"
 
+#include <cstdlib>
 #include <iostream>
 #include <sstream>
 #include <time.h>
diff -ru fuppes-0+svn578/src/lib/HTTP/HTTPServer.cpp fuppes-0+svn578-fixed/src/lib/HTTP/HTTPServer.cpp
--- fuppes-0+svn578/src/lib/HTTP/HTTPServer.cpp	2008-07-09 20:01:02.000000000 -0400
+++ fuppes-0+svn578-fixed/src/lib/HTTP/HTTPServer.cpp	2008-07-09 19:58:32.000000000 -0400
@@ -30,6 +30,7 @@
 #include "../Common/RegEx.h"
 #include "../DeviceSettings/DeviceIdentificationMgr.h"
 
+#include <cstdlib>
 #include <iostream>
 #include <sstream>
 #ifndef WIN32
diff -ru fuppes-0+svn578/src/lib/SSDP/UDPSocket.cpp fuppes-0+svn578-fixed/src/lib/SSDP/UDPSocket.cpp
--- fuppes-0+svn578/src/lib/SSDP/UDPSocket.cpp	2008-07-09 20:01:02.000000000 -0400
+++ fuppes-0+svn578-fixed/src/lib/SSDP/UDPSocket.cpp	2008-07-09 19:58:32.000000000 -0400
@@ -25,6 +25,7 @@
 #include "../SharedLog.h"
 #include "../SharedConfig.h"
 
+#include <cstring>
 #include <iostream>
 #include <sstream>
 
diff -ru fuppes-0+svn578/src/lib/Transcoding/FaadWrapper.cpp fuppes-0+svn578-fixed/src/lib/Transcoding/FaadWrapper.cpp
--- fuppes-0+svn578/src/lib/Transcoding/FaadWrapper.cpp	2008-07-09 20:01:02.000000000 -0400
+++ fuppes-0+svn578-fixed/src/lib/Transcoding/FaadWrapper.cpp	2008-07-09 19:58:32.000000000 -0400
@@ -46,6 +46,8 @@
 #include "../SharedLog.h" 
 #include "../SharedConfig.h" 
 
+#include <cstring>
+
 static int adts_sample_rates[] = {96000,88200,64000,48000,44100,32000,24000,22050,16000,12000,11025,8000,7350,0,0,0};
 
 int CFaadWrapper::adts_parse(int *bitrate, float *length)
diff -ru fuppes-0+svn578/src/lib/Transcoding/LameWrapper.cpp fuppes-0+svn578-fixed/src/lib/Transcoding/LameWrapper.cpp
--- fuppes-0+svn578/src/lib/Transcoding/LameWrapper.cpp	2008-07-09 20:01:02.000000000 -0400
+++ fuppes-0+svn578-fixed/src/lib/Transcoding/LameWrapper.cpp	2008-07-09 19:58:32.000000000 -0400
@@ -27,6 +27,8 @@
 
 #include "../SharedLog.h"
 #include "../SharedConfig.h"
+
+#include <cstring>
 #include <iostream>
 #include <sstream>
 
diff -ru fuppes-0+svn578/src/lib/Transcoding/PcmEncoder.cpp fuppes-0+svn578-fixed/src/lib/Transcoding/PcmEncoder.cpp
--- fuppes-0+svn578/src/lib/Transcoding/PcmEncoder.cpp	2008-07-09 20:01:02.000000000 -0400
+++ fuppes-0+svn578-fixed/src/lib/Transcoding/PcmEncoder.cpp	2008-07-09 19:58:32.000000000 -0400
@@ -23,6 +23,7 @@
 
 #include "PcmEncoder.h"
 
+#include <cstring>
 #include <iostream>
 using namespace std;
 
diff -ru fuppes-0+svn578/src/lib/Transcoding/TranscodingCache.cpp fuppes-0+svn578-fixed/src/lib/Transcoding/TranscodingCache.cpp
--- fuppes-0+svn578/src/lib/Transcoding/TranscodingCache.cpp	2008-07-09 20:01:02.000000000 -0400
+++ fuppes-0+svn578-fixed/src/lib/Transcoding/TranscodingCache.cpp	2008-07-09 19:58:32.000000000 -0400
@@ -34,6 +34,7 @@
 #include "LameWrapper.h"
 #endif
 
+#include <cstring>
 #include <iostream>
 #include <fstream>
 
diff -ru fuppes-0+svn578/src/lib/Transcoding/WavEncoder.cpp fuppes-0+svn578-fixed/src/lib/Transcoding/WavEncoder.cpp
--- fuppes-0+svn578/src/lib/Transcoding/WavEncoder.cpp	2008-07-09 20:01:02.000000000 -0400
+++ fuppes-0+svn578-fixed/src/lib/Transcoding/WavEncoder.cpp	2008-07-09 19:58:32.000000000 -0400
@@ -23,6 +23,7 @@
 
 #include "WavEncoder.h"
 
+#include <cstring>
 #include <iostream>
 using namespace std;
 
diff -ru fuppes-0+svn578/src/lib/libmain.cpp fuppes-0+svn578-fixed/src/lib/libmain.cpp
--- fuppes-0+svn578/src/lib/libmain.cpp	2008-07-09 20:01:02.000000000 -0400
+++ fuppes-0+svn578-fixed/src/lib/libmain.cpp	2008-07-09 19:58:32.000000000 -0400
@@ -23,6 +23,7 @@
 
 #include "../../include/fuppes.h"
 
+#include <cstring>
 #include <iostream>
 #include "Common/Common.h"
 #include "SharedConfig.h"
